runtime/ops/bootstrap.rs

```diff
@@ -39,6 +39,17 @@ pub struct SnapshotOptions {
   pub target: String,
 }
 
+impl Default for SnapshotOptions {
+  fn default() -> Self {
+    Self {
+      deno_version: "git".to_owned(),
+      ts_version: "n/a".to_owned(),
```

Comment:
These two values work for the __runtime_js_sources mode, but they are obviously not ideal. It's just a `Default` impl for `SnapshotOptions`, so it might not be a big deal.

Reply:
I would say that `deno_version` could be `dev` and `ts_version` is fine as it is. It really shows the leaky abstraction here that `runtime/` really depends on TS compiler being present...